### PR TITLE
CCD-1109 corrected area rmap + tpn

### PR DESCRIPTION
**Pull request by ddkauffman**

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -1,3 +1,11 @@
+11.9.1 (2022-02-21)
+===================
+
+Roman
+-----
+
+- corrected to area rmap to match updates to schema [#863]
```
Comment (eslavich) [inline CHANGES.rst:7]: Something is odd about the grammar here

File: crds/roman/specs/wfi_area.rmap [hunk 2]
```diff
@@ -2,18 +2,18 @@ header = {
...
     'observatory' : 'ROMAN',
     'parkey' : (('ROMAN.META.INSTRUMENT.DETECTOR', 'ROMAN.META.INSTRUMENT.OPTICAL_ELEMENT'), ('ROMAN.META.OBSERVATION.DATE', 'ROMAN.META.OBSERVATION.TIME')),
-    'sha1sum' : '6d600afa3482028ba926ef84fd80da8e806d121a',
-    'suffix' : 'pixelarea',
-    'text_descr' : 'pixelarea',
-    'tpn' : 'wfi_pixelarea.tpn',
+    'sha1sum' : 'b48381f4b3d3a66741eb58556ed2c136431bdb44',
+    'suffix' : 'AREA',
+    'text_descr' : 'AREA',
+    'tpn' : 'wfi_AREA.tpn',
```
Comment (eslavich) [inline crds/roman/specs/wfi_area.rmap:16]: These should follow the pattern of the other ref types with regard to case.  The suffix is lowercase `area`, the text description (which displays on the website) is capitalized `Area` and the tpn is lowercase `wfi_area.tpn` (this is name of the file below).

File: CHANGES.rst [hunk 3]
```diff
@@ -1,3 +1,11 @@
+11.9.1 (2022-02-21)
+===================
+
+Roman
+-----
+
+- corrected area rmap to match updates to schema [#863]
```
Comment (nden) [inline CHANGES.rst:7]: Is this entry in the right place?
I see it's under a section which was already released.

Comment (nden) [reply CHANGES.rst:7]: And below there's a previous unreleased version ??

**Review by eslavich — APPROVED** [review]
LGTM

**Review by nden — APPROVED** [review]

**Pull request merged**